app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] Please add a foreign key for posts, likes, and comments to set up the associations 

```ruby
has_many :posts, foreign_key: 'user_id'
has_many :comments, foreign_key: 'user_id'
has_many :likes, foreign_key: 'user_id'
```

Reply:
Thank you for the suggestion. I have successfully implemented the changes